src/main/java/uk/gov/pay/products/validations/RequestValidations.java

```diff
@@ -116,8 +115,4 @@
             return false;
         };
     }
-
-    public static Function<JsonNode, Boolean> isNotBoolean() {
-        return jsonNode -> !ImmutableList.of("true", "false").contains(jsonNode.asText().toLowerCase());
-    }
```

Comment:
Presumably this method is unused

Reply:
That's certainly what IntelliJ and `git grep` think